### ISSUE TITLE
Flag and config option to force build all aur packages in clean chroot

### ISSUE THREAD

**Issue by utrumo** [fosskers/aura#895]
Previously I used paru with --chroot flag always.

Now I tried aura and found that here you need to add each packet that I want to assemble in a similar way to config.toml in the chroot array.

It would be great if there was a flag that allowed all packets to be forcibly collected in clean chroot

**Comment by RubenKelevra** [comment]
Yeah, maybe a simple `chroot = yes|no|[package_list]` would be a good compromise?

**Comment by fosskers** [comment]
I knew that someone would request this, so let me say here that the current chroot-based building is considered experimental, hence the somewhat heavy-handed opt-in via a package list. It's not ideal, but I want to make sure there aren't any larger issues before activating it more widely.
